amgut/db/patches/0040.sql

```diff
@@ -255,3 +257,86 @@ CREATE TABLE pm.run_library_plate (
  );
 CREATE INDEX idx_run_library_plate_run_id ON pm.run_library_plate ( run_id );
 CREATE INDEX idx_run_library_plate_library_plate_id ON pm.run_library_plate ( library_plate_id );
+
+-- Add options for properties
+
+INSERT INTO pm.plate_type (name, cols, rows, notes)
+		VALUES ('96-well', 12, 8, 'Standard 96-well plate');
+
+INSERT INTO pm.extraction_robot (name)
+		VALUES ('HOWE_KF1'), ('HOWE_KF2'), ('HOWE_KF3'), ('HOWE_KF4');
+
+INSERT INTO pm.extraction_tool (name)
+		VALUES ('108379Z');
+
+INSERT INTO pm.processing_robot (name)
+		VALUES ('ROBE'), ('RIKE'), ('JERE'), ('CARMEN');
+
+INSERT INTO pm.tm300_8_tool (name)
+		VALUES ('208484Z'), ('311318B'), ('109375A'), ('3076189');
+
+INSERT INTO pm.tm50_8_tool (name)
+		VALUES ('108364Z'), ('311426B'), ('311441B'), ('409172Z');
+
+INSERT INTO pm.extraction_kit_lot (name)
+		VALUES ('PM16B11');
+
+INSERT INTO pm.master_mix_lot (name)
+		VALUES ('14459');
+
+INSERT INTO pm.water_lot (name)
+		VALUES ('RNBD9959');
+
+-- Migrate data from barcodes to pm
+
+INSERT INTO pm.sample (sample_id)
+		SELECT barcode FROM barcodes.barcode;
+
+UPDATE pm.sample SET barcode = sample_id;
+
+INSERT INTO pm.study (qiita_study_id, title)
```

Comment:
I think our previous discussions were: A study in plate mapper has its unique incremental study ID. The study can be imported from Qiita, in which case there is an additional qiita_study_id. Or, it can be non-Qiita (e.g., AG), in which case there is no qiita_study_id. In the very beginning you proposed to use positive number as Qiita study ID and negative number as customized study ID. During later discussions on GitHub we changed the plan and added the qiita_study_id column.